src/nbl/video/device_capabilities/device_features.json

```diff
@@ -980,7 +980,7 @@
         {
             "type": "bool",
             "name": "robustImageAccess",
-            "value": false,
+            "value": true,
```

Comment:
do not ever init the robustness features to default on, even if everyone has them